manifests/cf-manifest/spec/rds_broker_spec.rb

```diff
@@ -0,0 +1,111 @@
+      expect(rds_sg).to be
+      expect(rds_sg["rules"]).to eq([{
+        "protocol" => "tcp",
+        "destination" => terraform_fixture("aws_backing_service_cidr_all"),
+        "ports" => "5432",
+      }])
+    end
+
+    it "adds to default_running_security_groups" do
+      sgs = properties.fetch("cc").fetch("default_running_security_groups")
+      expect(sgs.length).to be > 1 # Ensure the default ones haven't been replaced
+      expect(sgs.last).to eq("rds_broker_instances")
+    end
+
+    it "adds to default_staging_security_groups" do
+      sgs = properties.fetch("cc").fetch("default_staging_security_groups")
+      expect(sgs.length).to be > 1 # Ensure the default ones haven't been replaced
+      expect(sgs.last).to eq("rds_broker_instances")
+    end
+  end
+
+  describe "service plans" do
+    let(:rds_broker_job) {
+      manifest.fetch("jobs").find {|j| j["name"] == "rds_broker_z1" }
+    }
+    let(:services) {
+      rds_broker_job.fetch("properties").fetch("rds-broker").fetch("catalog").fetch("services")
+    }
+    let(:all_plans) {
+      services.map {|s| s["plans"] }.flatten(1)
+    }
+
+    specify "all services have a unique id" do
+      all_ids = services.map {|s| s["id"] }
+      duplicated_ids = all_ids.select {|id| all_ids.count(id) > 1}.uniq
+      expect(duplicated_ids).to be_empty,
+        "found duplicate service ids (#{duplicated_ids.join(',')})"
+    end
+
+    specify "all services have a unique name" do
+      all_names = services.map {|s| s["name"] }
+      duplicated_names = all_names.select {|name| all_names.count(name) > 1}.uniq
+      expect(duplicated_names).to be_empty,
+        "found duplicate service names (#{duplicated_names.join(',')})"
+    end
+
+    specify "all plans have a unique id" do
+      all_ids = all_plans.map {|p| p["id"] }
+      duplicated_ids = all_ids.select {|id| all_ids.count(id) > 1}.uniq
+      expect(duplicated_ids).to be_empty,
+        "found duplicate plan ids (#{duplicated_ids.join(',')})"
+    end
+
+    specify "all plans have a unique name" do
+      all_names = all_plans.map {|p| p["name"] }
+      duplicated_names = all_names.select {|name| all_names.count(name) > 1}.uniq
+      expect(duplicated_names).to be_empty,
+        "found duplicate plan names (#{duplicated_names.join(',')})"
+    end
+
+    describe "postgres service" do
+      let(:pg_service) { services.find {|s| s["name"] == "postgres" } }
+      let(:pg_plans) { pg_service.fetch("plans") }
+
+      describe "plan rds_properties" do
+        shared_examples "all postgres plans" do
+          it { is_expected.to include("engine_version" => "9.5.2") }
+          it { is_expected.to include("storage_type" => "gp2") }
+          it { is_expected.to include("db_subnet_group_name" => terraform_fixture("rds_broker_dbs_subnet_group")) }
+          it { is_expected.to include("vpc_security_group_ids" => [terraform_fixture("rds_broker_dbs_security_group_id")]) }
+        end
+
```

Comment:
I'm not sure to see the value of these tests. The most important is already covered in the acceptante test. This only tests that spruce works and this adds code and properties to maintain.